### PR TITLE
Enable konflux on all release-next branches

### PR DESCRIPTION
**Pull request by pierDipi**
It works on eventing core, so enabling more repositories https://github.com/openshift-knative/eventing/pull/623

Also fixed error in https://github.com/openshift-knative/hack/actions/runs/10261906421

### CODE REVIEW

File: config/eventing.yaml [hunk 1]
```diff
@@ -27,8 +27,6 @@ config:
       - version: "4.15"
       - version: "4.12"
     release-v1.15:
-      konflux:
-        enabled: true
```
Comment (pierDipi) [inline config/eventing.yaml:31]: Removed 1.15 for now

File: pkg/action/update_action.go [hunk 2]
```diff
@@ -84,7 +84,7 @@ func UpdateAction(cfg Config) error {
 						},
 						"working-directory": fmt.Sprintf("./src/github.com/openshift-knative/hack/%s", r.RepositoryDirectory()),
 						"run": fmt.Sprintf(`set -x
-git remote add fork "https://github.com/serverless-qe/%s.git"
+git remote add fork "https://github.com/serverless-qe/%s.git" || true # ignore: already exists errors
```
Comment (pierDipi) [inline pkg/action/update_action.go:87]: Fix for https://github.com/openshift-knative/hack/actions/runs/10261906421

File: .github/workflows/release-generate-ci.yaml
Comment (pierDipi) [inline .github/workflows/release-generate-ci.yaml:1]: This is generated with `make generate-action`

**Review by creydr — COMMENTED** [review]
LGTM for eventing

@ReToCode / @skonto: also OK with serving part?

**Review by ReToCode — APPROVED** [review]
/lgtm
/approve


Yepp, let's see how that works 👍

**Pull request merged**